### PR TITLE
nift 3.0.0

### PR DESCRIPTION
**Pull request by n-ham**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Had trouble with lack of access permissions locally, then it was saying the bump commit already exists and need to do the PR manually..

### CODE REVIEW

File: Formula/nift.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nift < Formula
   desc "Cross-platform open source framework for managing and generating websites"
   homepage "https://nift.dev/"
-  url "https://github.com/nifty-site-manager/nsm/archive/v2.4.12.tar.gz"
-  sha256 "7a28987114cd5e4717b31a96840c0be505d58a07e20dcf26b25add7dbdf2668b"
+  url "https://github.com/nifty-site-manager/nsm/archive/v3.0.0.tar.gz"
+  sha256 "cdd8253745eca2970d9a986f0f3eb79ac090b7c4559471a3b0b58d20da58d721"
   license "MIT"
   revision 1
```
Comment (dawidd6) [inline Formula/nift.rb:7]:
```suggestion
```

File: Formula/nift.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Nift < Formula
   desc "Cross-platform open source framework for managing and generating websites"
   homepage "https://nift.dev/"
-  url "https://github.com/nifty-site-manager/nsm/archive/v2.4.12.tar.gz"
-  sha256 "7a28987114cd5e4717b31a96840c0be505d58a07e20dcf26b25add7dbdf2668b"
+  url "https://github.com/nifty-site-manager/nsm/archive/v3.0.0.tar.gz"
+  sha256 "cdd8253745eca2970d9a986f0f3eb79ac090b7c4559471a3b0b58d20da58d721"
```
Comment (chenrui333) [inline Formula/nift.rb:5]: looks like there is also a sha change, I am gonna take it into a separate PR.

**Comment by n-ham** [comment]
I just had to push a change due to GetObject needing to be GetObj with RapidJSON for Windows compatibility, my apologies but the sha has probably changed again..

**Comment by chenrui333** [comment]
@n-ham, no worries, I have included the updated sha in #131912 and shipped it out.

**Pull request closed without merging**